docs/scripts/api/buildInterfacesDocumentation.ts

```diff
@@ -33,9 +33,7 @@ interface ParsedProperty {
   typeStr: string;
 }
 
-const INTERFACES_WITH_DEDICATED_PAGES = [
-  // apiRef
-  'GridApi',
```

Comment:
Could it point to `GridApiPro` to prevent the 404 in https://deploy-preview-3648--material-ui-x.netlify.app/api/data-grid/grid-api/?

Reply:
I can add a redirection
And in the future we will indeed have to improve the doc to handle those renamed interfaces across plans.